python/unitytrainers/ppo/trainer.py

```diff
@@ -95,8 +95,18 @@ def __init__(self, sess, env, brain_name, trainer_parameters, training, seed):
 
         self.summary_writer = tf.summary.FileWriter(self.summary_path)
 
+        self.inference_run_list = [self.model.output, self.model.all_probs, self.model.value,
+                                   self.model.entropy, self.model.learning_rate]
+        if self.is_continuous_action:
+            self.inference_run_list.append(self.model.output_pre)
+        if self.use_recurrent:
+            self.inference_run_list.extend([self.model.memory_out])
```

Comment:
Has this been tested out with the hallway ?

Reply:
Yep.